<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leFrom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Преобразовать</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Из</string>
    </property>
    <widget class="QRadioButton" name="rBntFrom2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBntFrom10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBntFrom16">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>16</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>В</string>
    </property>
    <widget class="QRadioButton" name="rBntTo10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBntTo2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBntTo16">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>16</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="leFromDetailed2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFromDetailed10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFromDetailed16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
